lib/common.sh

```
@@ -88,19 +88,62 @@ function show_update_error {
   exit 1
 }
 
-# check if ansible return failure
+# check if ansible return failure on up
 # if failure, send to crash app
-function crash_test {
+function crash_test_up {
   RESULT=$?
   if [ $RESULT -ne 0 ]; then
-    /k2-crash-app/k2-crash-application $log_file
-    show_post_cluster_error 
+    # check the logs file exists
+    if [ -f "$log_file" ]; then
+      /usr/bin/k2-crash-app/k2-crash-application $log_file
```

I am unsure how great that my suggestion is though. You would still have to avoid calling `${k2-crash-application}` forever after, or call a wrapper which ignores errors. Part of me thinks you are on your own if you do not use a support k2-tools image. ¯\_(ツ)_/¯